pkg/deploy/config.go

```diff
@@ -245,16 +244,8 @@ func UnmarshalProviderMap(data string) (ProviderMap, error) {
 // RunConfigTest runs ConfigTest() if it is implemented on the interface
 func RunConfigTest(ctx context.Context, testable interface{}) error {
 	if tester, ok := testable.(gconfig.Tester); ok {
-		si := spinner.New(spinner.CharSets[14], 100*time.Millisecond)
```

Comment:
Does CLIO automatically format the global logger now so these look nice even when using zap.S().Info("message")

Reply:
Not automatically, but I used 

```
zap.ReplaceGlobals(clio.G())
```

in the entrypoint for `gdeploy` so that calls zap.S() are formatted by the `clio` formatter